tests/Microsoft.Identity.Test.Unit.net45/RequestsTests/IntegratedWindowsAuthUsernamePasswordTests.cs

```diff
@@ -297,6 +297,53 @@ public async Task AcquireTokenByIntegratedWindowsAuthTestAsync()
             }
         }
 
+        [TestMethod]
+        [TestCategory("IntegratedWindowsAuthAndUsernamePasswordTests")]
```

Comment:
Let's stop using TestCategory with the name of the class, it's pointless. Tests are already organized by class.